tests/rps_v2_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+require 'rps_v2'
+
+class RockPaperScissorsTest < Minitest::Test
+
+	def test_player_one_win
+		sample_game =  RockPaperScissorsGame.new
+		sample_game.set_p1_weapon("rock")
+		sample_game.set_p2_weapon("scissors")
+		@win_hash = {"paper" => ["scissors", "lizard"], "scissors" => ["rock", "spock"], "rock" => ["paper", "spock"], "lizard" => ["rock", "scissors"], "spock" => ["paper", "lizard"]}
+	  	assert_equal(@player1, sample_game.find_winner)
```

Comment:
It is **impossible** to access an instance variable except through an object's method. If you had a method in your `RockPaperScissorsGame` class like this:

```ruby
def get_player_1
  return @player1
end
```

Then you could run `sample_game.get_player_1`. And it would return the requested information.

But you should understand that it is **impossible** to simply ask, "What's `@player1`?" – it will just return `nil`.

So even if your tests are passing, you should investigate how/why. Because `@player1` is definitely `nil`.